pandas/tests/arrays/floating/test_astype.py

```diff
@@ -118,3 +118,19 @@ def test_astype_object(dtype):
     # check exact element types
     assert isinstance(result[0], float)
     assert result[1] is pd.NA
+
+
+@pytest.mark.parametrize(
```

Comment:
yeah can you move this to the base class to avoid this duplication

Reply:
you can use the data fixture

Reply:
Moved + rewritten with fixtures (though not 100% sure that what I did is what you asked)